cc_licenses/urls.py

```diff
@@ -25,25 +25,25 @@
 from licenses.views import branch_status, translation_status
 
 urlpatterns = [
-    url(r"^admin/", admin.site.urls),
+    re_path(r"^admin/", admin.site.urls),
     path("", TemplateView.as_view(template_name="home.html"), name="home"),
-    url(
+    re_path(
         r"status/(?P<id>\d+)/$",
         branch_status,
         name="branch_status",
     ),
-    url(
+    re_path(
         r"status/$",
         translation_status,
         name="translation_status",
     ),
-    url(r"licenses/", include("licenses.urls")),
+    path("", include("licenses.urls")),
```

Comment:
Just gut checking that this shouldn't be re_path? All other instances replace url with re_path, except this one.

I don't want to make the assumption that it is to do with what comes after (i.e. there is no (r" immediately after in this instance, but there is in all other ones).